lib/paginas/pagina_home.dart

```diff
@@ -0,0 +1,202 @@
+import "package:flutter/material.dart";
+//import ""; //Página Login
+//import ""; //Página Mensagens
+
+void main() {
+  runApp(Pagina_Home());
+}
+
+class Pagina_Home extends StatefulWidget {
+  const Pagina_Home({Key? key}) : super(key: key);
+
+  @override
+  Pagina_HomeEstado createState() => Pagina_HomeEstado();
+}
+
+class Pagina_HomeEstado extends State<Pagina_Home> {
+  int quantidade_utilizadores = 12;
+  //A quantidade de nomes inseridos dentro desta lista tem de estar de acordo com a variável quantidade_utilizadores
+  //Esta lista de utilizadores são exemplos, que vai ser aqui depois adicionar o código para ser lista de pedidos de amizades da base de dados
+  final List<String> utilizadores = [
+    "antonuolink",
+    "ToacyVaiMeDar20",
+    "MrEstudador",
+    "Afonso295",
+    "Cellbit",
+    "SigmaMale",
+    "5ugg4rM0mmy",
+    "AmanteN21DoAntonio",
+    "Bill_Gates",
+    "iShowSpeed",
+    "_Obama_",
+    "JohanLiebert2817"
+  ];

```

Comment:
![My answer to this code](https://i.giphy.com/media/v1.Y2lkPTc5MGI3NjExM2tvM2dqbTVwOXRneTR1eXpnaDdrNXg1OWhtcTZ6d3BsenNneGpwOSZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/xULW8N9O5WD32L5052/giphy.gif)